### PR TITLE
[ID] Update `osu!_Community_Meetings`

### PR DESCRIPTION
**Pull request by ZeroPyrozen**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/osu!_Community_Meetings/id.md [hunk 1]
```diff
@@ -6,11 +6,11 @@
 
 Saat ini, pertemuan komunitas osu! diketuai oleh [peppy](https://osu.ppy.sh/users/2), [Ephemeral](https://osu.ppy.sh/users/102335), dan [smoogipoo](https://osu.ppy.sh/users/1040328).
 
-Pertemuan komunitas osu! berikutnya akan diadakan pada **Minggu, 28 November 2021 pukul 01.00 UTC (08.00 WIB)**. Pertemuan diadakan setiap dua minggu dengan pengingat yang diumumkan melalui [akun Twitter peppy](https://twitter.com/ppy) sehari sebelumnya.
+Pertemuan komunitas osu! berikutnya akan diadakan pada **Sabtu, 11 Desember 2021 pukul 01.00 UTC (08.00 WIB)**. Pertemuan diadakan setiap dua minggu sekali dengan pengingat yang diumumkan melalui [akun Twitter peppy](https://twitter.com/ppy) sehari sebelumnya.
```
Comment (daud-ahn) [inline wiki/osu!_Community_Meetings/id.md:9]:
```suggestion
Pertemuan komunitas osu! berikutnya akan diadakan pada **Sabtu, 11 Desember 2021 pukul 21.00 UTC (04.00 WIB)**. Pertemuan diadakan setiap dua minggu sekali dengan pengingat yang diumumkan melalui [akun Twitter peppy](https://twitter.com/ppy) sehari sebelumnya.
```

**Review by daud-ahn — CHANGES_REQUESTED** [review]

**Comment by daud-ahn** [comment]
Apparently local time is Minggu 12 Desember 04.00 WIB

**Comment by ZeroPyrozen** [comment]
yeah I've included the local time on upcoming meeting but I keep the past meeting date as it is

**Review by hikikuman — APPROVED** [review]
already discussed in discord👍

**Review by TicClick — APPROVED** [review]

**Pull request merged**